src/Foundation/NSDictionary_2.cs

```diff
@@ -147,7 +147,28 @@ public TValue[] ObjectsForKeys (TKey[] keys, TValue marker)
 			return Runtime.GetNSObject<NSDictionary<TKey,TValue>> (_FromObjectsAndKeysInternal (objects.Handle, keys.Handle));
 		}
 
-		public static NSDictionary<TKey,TValue> FromObjectsAndKeys (TKey [] objects, TValue [] keys)
+		public static NSDictionary<TKey, TValue> FromObjectsAndKeys (TValue [] objects, TKey [] keys, nint count)
```

Comment:
I wondered too but when I tried to change the new unit test to use `<NSObject,NSObject>` (with `NSObject` arrays) it did not cause issue at build time.